.github/workflows/release.yml

```diff
@@ -135,4 +135,48 @@ jobs:
           generateReleaseNotes: true
           token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ github.event.inputs.releaseTag }}
-          prerelease: ${{ github.event.inputs.productionRelease && github.event.inputs.productionRelease == 'false' }}
\ No newline at end of file
+          prerelease: ${{ github.event.inputs.productionRelease && github.event.inputs.productionRelease == 'false' }}
+
+  release-debian:
+    name: Build for Linux
+    runs-on: ubuntu-latest
```

Comment:
You are creating one instance of ubuntu, building rust for linux there, regular binary, and now creating another Ubuntu machine to create deb on it? Don't. Use the previous `Build for Linux`, and add the step. Check the video out please.